tx-source/dash_core.php

```diff
@@ -725,6 +728,7 @@
 
 $Definition['Page Color'] = 'Page Color';
 $Definition['Page Views'] = 'Page Views';
+$Definition['Page Views'] = 'Pageview ratio representing a deflected case';
```

Comment:
This is probably not right ? :)

Reply:
🤦‍♀️ Ooooops, thanks for catching that! Should be fixed now in b1a3fa4 😄